### ISSUE TITLE
Problem with record and play wav files on pi 

### ISSUE THREAD

**Issue by amirush786** [dwelch67/raspberrypi#46]
Hello,
I want to record and play a wav file at the same time with the setup (pi + 6mic tool +jbl speaker) so I have to play a modulated sound from the speaker and record that sound from the 6mic tool which is connected to pi. So I am ablt to do that but the wav file that's generated has some hissshhhh kinds sound instead of the recorded expected sound which shoukd be recorded. Can you help me fix this. Don't know it might be issue with the Alsa mixer. @HinTak

**Comment by HinTak** [comment]
FWIW, you have the square 4-mics device! Follow the 4-mics device instructions, not 6-mics or the linear-4-mics instructions! And, you really need somebody sitting in front of the device with you, so you need a local pi/linux person, not online, really. So cannot help you any further.

**Comment by amirush786** [comment]
> FWIW, you have the square 4-mics device! Follow the 4-mics device instructions, not 6-mics or the linear-4-mics instructions! And, you really need somebody sitting in front of the device with you, so you need a local pi/linux person, not online, really. So cannot help you any further. 

Yes Its recording and playing at the same time thabkyoy very much for that. But now the issue with the Alsa mixer the wav file after doing arecord that file still plays shhhhh sound I don't know why so I will explain what I did. I did Alsamixer command and it's shows me this. 


So after that I typed in F6 to change my card to seeed card.

Then is scroll down to seed card and hit enter. So I see this. 

Then I press f4 and I see this so I don't know where should I see that capture is high.
This is the last thing that messing with me.

![20210822_200626.jpg](https://user-images.githubusercontent.com/31735000/131002419-3e260acb-bc24-4334-b70e-9a004adb351d.jpg)

![20210822_200634.jpg](https://user-images.githubusercontent.com/31735000/131002464-ba1172ae-c3a3-4089-8862-b87dcea81400.jpg)

![20210822_201414.jpg](https://user-images.githubusercontent.com/31735000/131002551-99c04780-3174-4314-88cc-a67ada1648a0.jpg)

**Comment by C-Elegans** [comment]
Umm, are you sure this is the right repository for this issue? As far as I can tell, this repository has nothing to do with sound on Linux.

**Comment by amirush786** [comment]
> Umm, are you sure this is the right repository for this issue? As far as I can tell, this repository has nothing to do with sound on Linux. 

The issue is with the pi Alsamixer do I don't which should be the right one to hit on so I used pi repo.

**Comment by C-Elegans** [comment]
> The issue is with the pi Alsamixer do I don't which should be the right one to hit on so I used pi repo.

Ah. In that case, might I recommend posting your issue either on the alsa github https://github.com/alsa-project or the alsa mailing list https://mailman.alsa-project.org/mailman/listinfo/alsa-devel?

This repository is about bare metal programming (C and assembly with no operating system) on the raspberry pi, you're probably not going to find the help you need here unless you get very lucky.

Hope you get your issue solved.
